src/pages-chibi/implementations/anicrush/main.ts

```diff
@@ -0,0 +1,58 @@
+import { PageInterface } from '../../pageInterface';
+
+export const anicrush: PageInterface = {
+  name: 'Anicrush',
+  type: 'anime',
+  domain: 'https://anicrush.to',
+  languages: ['English'],
+  urls: {
+    match: ['*://anicrush.to/*'],
+  },
+  search: 'https://anicrush.to/search?keyword={searchtermPlus}',
+  sync: {
+    isSyncPage($c) {
+      return $c.url().urlPart(3).equals('watch').run();
+    },
+    getTitle($c) {
+      return $c.querySelector('div.main h2').text().trim().run();
+    },
+    getIdentifier($c) {
+      return $c.url().urlPart(4).run();
+    },
+    getOverviewUrl($c) {
+      return $c.querySelector('div.other-items a').getAttribute('href').urlAbsolute().run();
+    },
+    getEpisode($c) {
+      return $c.url().urlParam('ep').number().run();
+    },
+  },
+  overview: {
+    isOverviewPage($c) {
+      return $c.url().urlPart(3).equals('detail').run();
+    },
+    getTitle($c) {
+      return $c.querySelector('div.main h2').text().trim().run();
+    },
+    getIdentifier($c) {
+      return $c.url().urlPart(4).run();
```

Comment:
if the function in sync and overview is the same you can call the other function to remove duplicate code
https://github.com/MALSync/MALSync/blob/master/src/pages-chibi/implementations/animeav1/main.ts#L45 like that

Reply:
could also be done for title